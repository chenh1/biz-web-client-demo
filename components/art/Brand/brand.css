.brand {
  -webkit-text-stroke: 2px #41ABFF;
  -webkit-text-fill-color: #0BEFFE;
  text-shadow: 3px 3px 0px  #41ABFF;
  letter-spacing: -3px;
}

.brand.mono {
  -webkit-text-stroke: 2px #F3F4F6;
  -webkit-text-fill-color: rgb(148 163 184);
  text-shadow: 3px 3px 0px  #F3F4F6;
}

.brand .segment-2,
.brand .segment-3,
.brand .segment-4 {
  margin-left: -2px;
}