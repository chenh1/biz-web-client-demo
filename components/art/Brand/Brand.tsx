"use client";
import React from "react";
import './brand.css';

interface BrandProps {
  mono?: boolean; 
}

export const Brand: React.FC<BrandProps> = ({ mono = false }) => {
  return (
    <div className={`brand ${mono ? 'mono' : ''} font-brand text-5xl`}>
      <span className="">ba</span>
      <span className="segment-2">yba</span>
      <span className="segment-3">r</span>
      <span className="segment-4">oo</span>
    </div>
  );
};