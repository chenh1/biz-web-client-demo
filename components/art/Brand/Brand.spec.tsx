import { Brand } from "./Brand";
import { render } from '@testing-library/react';
import { JSDOM } from "jsdom"
const dom = new JSDOM()
global.document = dom.window.document
global.window = dom.window


it('renders company branding', async () => {
  const { getByTestId, asFragment } = render(
    <Brand/>
  );

  expect(asFragment()).toMatchSnapshot();
});