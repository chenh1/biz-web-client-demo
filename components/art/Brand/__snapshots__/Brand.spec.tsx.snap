// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders company branding 1`] = `
<DocumentFragment>
  <div
    class="brand  font-brand text-5xl"
  >
    <span
      class=""
    >
      ba
    </span>
    <span
      class="segment-2"
    >
      yba
    </span>
    <span
      class="segment-3"
    >
      r
    </span>
    <span
      class="segment-4"
    >
      oo
    </span>
  </div>
</DocumentFragment>
`;
